addons/website_slides_survey/views/website_profile.xml

```diff
@@ -14,20 +14,15 @@
      <template id="display_certificate">
         <t t-if="certificates">
             <div class="row">
-                <div class="col-lg-6 o_wprofile_slides_card" t-foreach="certificates" t-as="certificate">
-                    <div class="card-body o_wprofile_slides_certif_card_body o_wprofile_bb_grey">
-                        <div class="row">
-                            <div><img class="o_wprofile_img_small" t-attf-src="/web/image/slide.slide/#{certificate.slide_id.id}/image_small"/></div>
-                            <div class="col-10 o_wprofile_slides_title">
-                                <span class="font-weight-bold" t-field="certificate.survey_id.title"/>
-                            </div>
-                            <div class="float-right" t-if="user.id == uid">
-                                <span class="row">Score : <b t-field="certificate.quizz_score"/>%</span>
-                                <div class="row">
-                                    <a role="button"
-                                        t-att-data-survey_id="certificate.survey_id.id"
-                                        class="o_wslides_survey_download_certification"
-                                        href="#">
+                <div class="col-12 col-lg-6" t-foreach="certificates" t-as="certificate">
+                    <div class="card mb-2">
+                        <div class="card-body o_wprofile_slides_course_card_body p-0 d-flex">
+                            <div class="pl-5 pr-4 rounded-left" t-attf-style="background-image:url(/web/image/slide.slide/#{certificate.slide_id.id}/image_medium); background-position: center"/>
```

Comment:
isn't it a bootstrap class instead of the inline style ?

Reply:
comes from SRI's design for Course card in profile page. Needed to copy the exact behaviour to have aligned design between every card with image.